fe/fe-core/src/main/java/com/starrocks/alter/AlterMVJobExecutor.java

```diff
@@ -152,6 +153,13 @@ public Void visitModifyTablePropertiesClause(ModifyTablePropertiesClause modifyT
             properties.remove(PropertyAnalyzer.PROPERTY_MV_ENABLE_QUERY_REWRITE);
         }
 
+        // warehouse
+        if (properties.containsKey(PropertyAnalyzer.PROPERTIES_WAREHOUSE)) {
+            String warehouseName = properties.remove(PropertyAnalyzer.PROPERTIES_WAREHOUSE);
```

Comment:
‌Why do we need to use the remove operation to obtain the warehouseName?

Reply:
Because of the code structure, this is just a continuation of the original design. The original logic is to gradually delete the properties in properties, and finally determine whether there is any unprocessed content.